maquetado/application.css

```diff
@@ -0,0 +1,22 @@
+body {
```

Comment:
There is no need to push this to the repository. This file is created by the sass-cli by compiling the ```.scss``` file but you only need the scss. Add this to the ```.gitignore``` file, as well as the ```.sass-cache```. The ```.gitignore``` file should look something like this: 
```
.sass-cache/*
*.css
```